src/Form/FormAnalyzer.js

```diff
@@ -160,11 +183,7 @@ class FormAnalyzer {
         this.evaluatePageTitle()
         this.evaluatePageHeadings()
         // Check for submit buttons
-        const buttons = document.querySelectorAll(`
-                button[type=submit],
-                button:not([type]),
-                [role=button]
-            `)
+        const buttons = document.querySelectorAll(this.matching.cssSelector('SUBMIT_BUTTON_SELECTOR'))
```

Comment:
`SUBMIT_BUTTON_SELECTOR` doesn't have `button:not([type])` which this used to use -- is that an issue?

Reply:
Good spot. Let me check that.

Reply:
No, it's fine as is. We have `button:not([role=switch]):not([role=link])` which is more comprehensive and we have more checks in `evaluateElement`.